tetris.py

```diff
@@ -69,46 +69,51 @@ def run_gui():
     new_game_button3 = tkinter.Button(sidebar, text="start")
     new_game_button3.grid(sticky="n")
 
-    tetris_gui = TetrisGUI(game_speed, game_canvas, topbar_board)
+    small_board = Board(topbar_canvas, 4, 2, D_GREY, (2, 1), None)
+    main_board = Board(game_canvas, game_width, game_height, GREY, (int(game_width / 2), -2), small_board)
+
+    main_board.current_block_mover()
+
+    tetris_gui = TetrisGUI(game_speed, main_board)
 
     root.bind("<Left>", tetris_gui.move_block_left)
     root.bind("<Right>", tetris_gui.move_block_right)
     root.bind("<Up>", tetris_gui.rotate_block)
 
-    tetris_gui.draw_board(game_canvas, game_width, game_height, GREY)
-    tetris_gui.draw_board(topbar_board, 4, 2, D_GREY)
-    tetris_gui.draw_block()
-    tetris_gui.move_block()
-
     root.title("Tetris – by The Philgrim, Arrinao, and Master Akuli")
     # root.iconphoto(False, tkinter.PhotoImage(file=image_name.png")) TODO: INSERT LATER
     root.mainloop()
 
-
-class TetrisGUI:
-    def __init__(self, speed, canvas, canvas_small):
-        self.speed = speed
+class Board:
+    def __init__(self, canvas, width, height, outline_color, current_block_center, small_board):
         self.canvas = canvas
-        self.canvas_small = canvas_small
-        self.rect_size = 25
-        self.tetris_game = TetrisGame()
-        self.start_time = time.time()
+        self.width = width
+        self.height = height
+        self.outline_color = outline_color
+        self.landed_blocks = {}
+        self.current_block_center = current_block_center
+        self.block_letter = random.choice(block_letter)
+        self.rotate_counter = 0
+        self.draw_board()
+        self.small_board = small_board
```

Comment:
It's a bit weird how `self.small_board = small_board` is between `draw_board` and `draw_block`. Maybe put it above `draw_board`?